src/config.rs

```diff
@@ -28,8 +28,43 @@ pub struct Config {
     pub flatpak: FlatpakConfig,
     #[serde_inline_default(Config::default().vscode)]
     pub vscode: VsCodeConfig,
+    pub yarn: YarnConfig,
 }
 
+macro_rules! apply_configs {
```

Comment:
I think we can get each backend's config using the existing `apply_backends!` macro by doing `config.$lower_backend`.